oxidation/client/package.json

```diff
@@ -5,11 +5,14 @@
     "type": "module",
     "scripts": {
         "postinstall": "cd electron && npm install",
-        "dev": "vite",
+        "dev": "vite --port 8080",
```

Comment:
The vite server port is also configured in `vite.config.ts`

Reply:
Yep but it is...weird. If it set the option `--port 8080`, the server starts on port 8080 and everything seems to work. If I set the port in vite.config, it starts on port 8081 because for some reason, it detects the port 8080 as being already in use. I can force `strictPort: true` in vite.config, but then the server doesn't start because it is not able to bind on port 8080.